tests/journal/balance-assertions.test

```diff
@@ -136,3 +136,140 @@ hledger -f - stats
 # >>> /Transactions/
 # >>>2
 # >>>=0
+
```

Comment:
I think it would be beneficial, I you changed the docs, because you see the whole picture and this also helps check again, if everything works as expected.